odl/operator/default_ops.py

```diff
@@ -754,15 +756,17 @@ class ConstantOperator(Operator):
         ``ConstantOperator(y)(x) == y``
     """
 
-    def __init__(self, vector, domain=None):
+    def __init__(self, vector, domain=None, range=None):
         """Initialize a new instance.
 
         Parameters
         ----------
         vector : `LinearSpaceElement`
             The constant space element to be returned.
         domain : `LinearSpace`, optional
-            Domain of the operator. Default : ``vector.space``.
+            Domain of the operator. Default: ``vector.space``
+        range : `LinearSpace`, optional
```

Comment:
How can range be needed? I suppose you changed `vector` to mean something like "`constant`"? Then this deserves a naming change.